[2023-09-19 07:52:44,868] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T01:27:00+00:00 [queued]>
[2023-09-19 07:52:45,130] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T01:27:00+00:00 [queued]>
[2023-09-19 07:52:45,131] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:52:45,132] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2023-09-19 07:52:45,132] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:52:45,386] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2023-09-19T01:27:00+00:00
[2023-09-19 07:52:45,499] {standard_task_runner.py:52} INFO - Started process 1388 to run task
[2023-09-19 07:52:45,708] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', '2023-09-19T01:27:00+00:00', '--job-id', '208', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_pipeline.py', '--cfg-path', '/tmp/tmp07q6hamk', '--error-file', '/tmp/tmpi4ivwygi']
[2023-09-19 07:52:45,709] {standard_task_runner.py:77} INFO - Job 208: Subtask fetch_weather_data
[2023-09-19 07:52:46,710] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T01:27:00+00:00 [running]> on host 92beb7708765
[2023-09-19 07:52:47,628] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data_pipeline
AIRFLOW_CTX_TASK_ID=fetch_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-19T01:27:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-19T01:27:00+00:00
[2023-09-19 07:52:48,873] {python.py:151} INFO - Done. Returned value was: True
[2023-09-19 07:52:49,415] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, execution_date=20230919T012700, start_date=20230919T075244, end_date=20230919T075249
[2023-09-19 07:52:50,014] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2023-09-19 07:52:50,260] {local_task_job.py:151} INFO - Task exited with return code 0
